fvtest.conf

```diff
@@ -0,0 +1,26 @@
+agentAddress udp:16161,udp6:[::]:16161
+rocommunity clearwater
+
+view clearwater included .1
+
+# SNMP system OIDs
+view clearwater included .1.3.6.1.2.1.1.1.0
```

Comment:
I don't think you need most of these lines - `view clearwater included .1` should do this.